lib/components/FormListItem.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const componentsRegistry = milo.registry.components;
+const FORM_LIST_CHANGE_MESSAGE = 'formlistchange';
+
+module.exports = milo.createComponentClass({
+    className: 'MLFormListItem',
+    facets: {
+        container: undefined,
+        data: {
+            get: MLFormListItem_get,
+            set: MLFormListItem_set,
+            del: MLFormListItem_del,
+            event: FORM_LIST_CHANGE_MESSAGE
+        },
+        item: undefined,
+        template: {
+            template: '\
```

Comment:
Would backticks allow you to avoid the annoying `\` needed for multiline strings when using `'`?

Reply:
Yeah we could probably update milo at some point to allow them but not an issue for this RP.